Cannot Resolve 'elk.bundled' While Building the App

<!--
PLEASE HELP US PROCESS GITHUB ISSUES FASTER BY PROVIDING THE FOLLOWING INFORMATION.

ISSUES MISSING IMPORTANT INFORMATION MAY BE CLOSED WITHOUT INVESTIGATION.
-->

## I'm submitting a...
<!-- Check one of the following options with "x" -->
<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Current behavior
<!-- Describe how the issue manifests. -->
Getting this error message while building the app.
<pre><code>
Module not found: Error: Can't resolve 'elkjs/lib/elk.bundled' in '...\node_modules\reaflow\dist'
Did you mean 'elk.bundled.js'?
BREAKING CHANGE: The request 'elkjs/lib/elk.bundled' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
</code></pre>

## Expected behavior
<!-- Describe what the desired behavior would be. -->
Should just build the app.

## Minimal reproduction of the problem with instructions
<!--
For bug reports please provide the *STEPS TO REPRODUCE* and if possible a *MINIMAL DEMO* of the problem via

* Step 1
* Step 2
* Step 3

-->

1. `npx create-react-app my-app`
2. In my-app folder, `npm i reaflow`
3. Put the example in readme file into app.js
4. `npm run build`

## What is the motivation / use case for changing the behavior?
<!-- Describe the motivation or the concrete use case. -->
I'm using OpenShift to build the app and it uses package.json to install all the npm packages. So I can't just go into the node module and manually fix it. I have tried manipulating webpack config using craco, but it introduces more problems than resolving this bug.

## Environment

<pre><code>
Libs:
- react version: 18.2.0
<!-- Check whether this is still an issue in the most recent React version -->
 
For Tooling issues:
- Node version: 18.15.0
- Platform:  Windows
- Webpack: 5.88.2

Hi, I have the same issue here, the error log is bellow. so I report here, hope it could help.

```
ERROR in ./node_modules/reaflow/dist/index.js 18:0-40
Module not found: Error: Can't resolve 'elkjs/lib/elk.bundled' in '.../node_modules/reaflow/dist'
Did you mean 'elk.bundled.js'?
BREAKING CHANGE: The request 'elkjs/lib/elk.bundled' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

my webpack version is 5.57.0.

--- a/src/layout/elkLayout.ts
+++ b/src/layout/elkLayout.ts
@@ -1,5 +1,5 @@
 import { EdgeData, NodeData } from '../types';
-import ELK, { ElkNode } from 'elkjs/lib/elk.bundled';
+import ELK, { ElkNode } from 'elkjs/lib/elk.bundled.js';
 import PCancelable from 'p-cancelable';
 import { formatText, measureText } from './utils';
 

